device/arista/x86_64-arista_7280dr3a_36/Arista-7280DR3A-36/0/buffers_defaults_t2.j2

```diff
@@ -0,0 +1,46 @@
+{%- set default_cable = '300m' %}
+
+{%- set ports2cable = {
+        'torrouter_server'       : '300m',
+        'leafrouter_torrouter'   : '300m',
+        'spinerouter_leafrouter' : '2000m',
+        'regionalhub_spinerouter': '120000m',
+        'aznghub_spinerouter'    : '120000m'

```

Comment:
@arista-hpandya please refer following PR for additional changes - 
https://github.com/sonic-net/sonic-buildimage/pull/22427